### ISSUE TITLE
Can't distinguish between code languages while write by Markdown

### ISSUE THREAD

**Issue by chogenfran** [aramk/crayon-syntax-highlighter#427]
![image](https://user-images.githubusercontent.com/9606502/35469925-8c47854e-037a-11e8-9763-0b442abd2104.png)
![image](https://user-images.githubusercontent.com/9606502/35469930-98741328-037a-11e8-9b2b-168a5f19c19c.png)

using JP Markdown,Markdown Editor,WP Editor.md make no difference

**Comment by luispuerto** [comment]
This happens to me also... did you find any workaround?

**Comment by chogenfran** [comment]
Yes,and the solution is [here](https://www.jamesheblog.com/%E7%BC%96%E7%A8%8B%E6%9D%82%E8%B0%88/%E8%A7%A3%E5%86%B3wp%E4%B8%ADmarkdown%E7%BC%96%E8%BE%91%E5%99%A8%E4%B8%8Ecrayon-syntax-highlighter%E7%9A%84%E5%86%B2%E7%AA%81/)

**Comment by luispuerto** [comment]
Problem is… I don’t use the wp-editormd. I just use jetpack. 

Any idea?
